.github/workflows/publish.yml

```diff
@@ -2,23 +2,36 @@ name: Build & Publish
 
 on:
   pull_request:
-    branches: [main, "*postfix"]
+    branches: [main, "version-*"]
```

Comment:
Adding this in preparation for making the 3.9.* branch(es). Therefore if and when we need to release a new 3.9 version for a bug fix, we can still build and publish versions from branches named `version-3.9.2`, `version-3.10.1`, etc.